### DISCUSSION TITLE
How can I get Flowise to Access a Fine-Tuned ChatGPT 4o mini model?

### DISCUSSION THREAD

**Discussion by pcaspa** [FlowiseAI/Flowise#2896] [Q&A]
Hi,

I've implemented a few chatbot with Flowise but I'd like to test the Fine Tuning feature now available with ChatGPT 4o-mini.  How can I get Flowise to Access a Fine-Tuned ChatGPT 4o mini model?

Thanks

**Comment by yavisht** [comment]
@pcaspa 

Once you have your fine-tuned model ready, you can setup a flow using the ChatOpenAI Custom chat model.

<img width="372" alt="image" src="https://github.com/user-attachments/assets/ca7947b7-3e51-4c8b-a808-72eebab25a54">

Once your fine tuned model is ready, just add your generated model name.

<img width="854" alt="Screenshot 2024-08-29 at 1 49 17 PM" src="https://github.com/user-attachments/assets/c205e8fd-6c5a-4691-b851-559c1e4df310">
